src/prasad/Findiftwostrareindentical.java

```diff
@@ -0,0 +1,34 @@
+package prasad;
+
+public class Findiftwostrareindentical {
+	
+	boolean scanStr(String input1 , String input2) {
+		boolean flag = false;
+		int count=0;
+		for(int index=0 ; index<input1.length(); index++) {
+			if(input1.length()!=input2.length() || input1.charAt(index)!=input2.charAt(index))
+				break;
+			
+			else if(input1.charAt(index)== input2.charAt(index)) {
```

Comment:
you can put condition  != condition and flag = false/true and then check flag value. Accordingly flag value print message,
Enhancement

Reply:
Corrected the class please check